pkgs/applications/window-managers/wayfire/default.nix

```diff
@@ -8,6 +8,7 @@
 , wf-config
 , cairo
 , doctest
+, doxygen
```

Comment:
Hmmm, how is this used? I got lost exploring the many subprojects, so far I only see this [in wf-utils as an optional dep](https://github.com/WayfireWM/wf-utils/blob/15f8e16721585ae3eaf278ba71d7064237eb23f5/meson.build#L107) and it is used in a [`run_target`](https://mesonbuild.com/Reference-manual_functions.html#run_target), which we don't run right now

Reply:
It may be because the wlroots 0.16 version was used before, which caused the wlroots(0.17) in submodule to start compiling, which is actually not needed.